rdt-open-reader/src/main/java/io/ona/rdt/widget/RDTBarcodeFactory.java

```diff
@@ -86,7 +86,7 @@ protected void moveToNextStep() {
     }
 
     private boolean isRDTExpired(Date date) {
-        return date == null ? true : new Date().after(date);
+        return date != null && new Date().after(date);
```

Comment:
can we add a test for this?

Reply:
That is covered [here](https://github.com/onaio/rdt-standard/blob/dcb9b4c804c2a13d5534efda9334d1f0444691c0/rdt-open-reader/src/test/java/io/ona/rdt/widget/RDTBarcodeFactoryTest.java#L110) and [here](https://github.com/onaio/rdt-standard/blob/dcb9b4c804c2a13d5534efda9334d1f0444691c0/rdt-open-reader/src/test/java/io/ona/rdt/widget/RDTBarcodeFactoryTest.java#L116).

Reply:
can you please add a test that fails with the old code and passes with the new code

Reply:
Done.